optaweb-vehicle-routing-backend/src/main/docker/Dockerfile.jvm

```diff
@@ -7,21 +7,21 @@
 #
 # Then, build the image with:
 #
-# docker build -f src/main/docker/Dockerfile.jvm -t quarkus/quarkus-jvm .
+# docker build -f src/main/docker/Dockerfile.jvm -t quarkus/optaweb-vehicle-routing-jvm .
 #
 # Then run the container using:
 #
-# docker run -i --rm -p 8080:8080 quarkus/quarkus-jvm
+# docker run -i --rm -p 8080:8080 quarkus/optaweb-vehicle-routing-jvm
 #
 # If you want to include the debug port into your docker image
-# you will have to expose the debug port (default 5005) like this :  EXPOSE 8080 5050
+# you will have to expose the debug port (default 5005) like this :  EXPOSE 8080 5005
 #
 # Then run the container using :
 #
-# docker run -i --rm -p 8080:8080 -p 5005:5005 -e JAVA_ENABLE_DEBUG="true" quarkus/quarkus-jvm
+# docker run -i --rm -p 8080:8080 -p 5005:5005 -e JAVA_ENABLE_DEBUG="true" quarkus/optaweb-vehicle-routing-jvm
 #
 ###
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3 
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.4
```

Comment:
Are there some particular reason to upgrade to 8.4? 
If it is possible I would like to suggest staying on 8.3.

Reply:
This is the result of running
```
$ mvn io.quarkus.platform:quarkus-maven-plugin:2.3.0.Final:create \
-DprojectGroupId=org.optaweb.vehiclerouting \
-DprojectArtifactId=optaweb-vehicle-routing \
-DprojectVersion=8.14.0-SNAPSHOT
```
So `ubi-minimal:8.4` is the base image version for Dockerfiles that are part of a fresh project bootstrapped with the Quarkus Maven plugin 2.3.0.Final.

The question is, why would we want to downgrade it from 8.4 to 8.3?

Reply:
It is not a problem, just the hack that we have to avoid Docker pull limit is made in the way that you need to explicitly mention the image. Any way I can update the tests and it should be fine